src/schema/schema.c

```diff
@@ -40,25 +40,25 @@ unsigned short Schema_IndexCount(const Schema *s) {
 }
 
 Index *Schema_GetIndex(const Schema *s, Attribute_ID *attribute_id, IndexType type) {
-	Index *idx = NULL;
```

Comment:
Should we `ASSERT(s->index != NULL || s-> fulltextIdx != NULL);`

Reply:
It is valid for no index to exist and for this function to return NULL; this option is required by the shared update routine that checks whether an update necessitates reindexing.

Reply:
👍